herbert_core/classes/MPIFramework/parallel_worker.m

```diff
@@ -175,11 +180,18 @@
         % something wrong with the code. We can not process interrupt
         % properly, but filebased framework should still be
         % available.
-        if DO_LOGGING; log_input_message_exception_caught();  end
-        err_mess = sprintf('job N%s failed. Error during job initialization %s:',...
-            control_struct.job_id,ME.message);
-        fbMPI.send_message(0,FailedMessage(err_mess,ME));
-        break;
+
+        if ~strcmp(ME.identifier,'MESSAGE_FRAMEWORK:canceled')  
```

Comment:
possible typo: `MESSAGE_FRAMEWORK:canceled` or `MESSAGE_FRAMEWORK:cancelled` ?

Reply:
no -- its indeed American spelling -- it in so many places, that better leave it alone.

Reply:
just wanted to check